packages/material-ui/src/styles/adaptV4Theme.test.js

```diff
@@ -2,183 +2,265 @@ import { expect } from 'chai';
 import adaptV4Theme from './adaptV4Theme';
 
 describe('adaptV4Theme', () => {
-  it("moves props to components' defaultProps", () => {
-    const theme = {
-      props: {
-        MuiButton: {
-          disabled: true,
+  describe('theme.components', () => {
```

Comment:
Git is showing this changes very badly 😢 I've just wrapped the tests related to the components section in this describe block.

Reply:
You can [hide whitespace from the diff](https://github.blog/2018-05-01-ignore-white-space-in-code-review/) which makes it much more readable: https://github.com/mui-org/material-ui/pull/22396/files/1bd4d245c201010fb90499a3e2e85c8d82a5d52d?diff=unified&w=1

Reply:
Wow, that's brilliant. This needs to be tweeted 😁

Reply:
Wow. Why isn't this default view I wonder.. Thanks for sharing it @eps1lon